src/coordination/azure/Akka.Coordination.Azure/AzureLease.cs

```diff
@@ -95,25 +95,25 @@ public override async Task<bool> Release()
             try
             {
                 if(_log.IsDebugEnabled)
-                    _log.Debug("Releasing lease");
+                    _log.Debug("Releasing lease [{0}] for [{1}]", _leaseName, _settings.OwnerName);
                 var result = await _leaseActor.Ask(LeaseActor.Release.Instance, _timeout);
                 switch (result)
                 {
                         case LeaseActor.LeaseReleased:
                             return true;
                         case LeaseActor.InvalidReleaseRequest:
-                            _log.Info("Tried to release a lease that is not acquired");
+                            _log.Info("Tried to release a lease [{0}] for [{1}] that is not acquired", _leaseName, _settings.OwnerName);
```

Comment:
Change all log and exception messages to log the name and the owner of the lease to make it easier to trace lease ownership